libs/partners/anthropic/langchain_anthropic/llms.py

```diff
@@ -375,9 +374,11 @@ async def _astream(
 
     def get_num_tokens(self, text: str) -> int:
         """Calculate number of tokens."""
-        if not self.count_tokens:
-            raise NameError("Please ensure the anthropic package is loaded")
-        return self.count_tokens(text)
+        raise NotImplementedError(
```

Comment:
to make this nonbreaking could we just call the new endpoint with a single humanmessage and subtract a constant number of standard tokens. (for e.g. `Human: `, and print a warning?

Reply:
Older models like claude 2 not supported unfortunately: https://docs.anthropic.com/en/docs/build-with-claude/token-counting#supported-models